_translator-files/po/de/Installation-for-Windows.po

```diff
@@ -69,19 +70,19 @@ msgstr "**Downloade und installiere einen ASIO-Treiber**. Wir empfehlen, eine So
 #: ../wiki/en/Installation-for-Windows.md:18
 #, no-wrap
 msgid "[Download Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows }}){: .button}\\\\\n"
-msgstr "[Downloade Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows }}){: .button}\\\\\n"
+msgstr "[Download Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows }}){: .button}\\\\\n"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:19
 #, no-wrap
 msgid "**Mirror 2:** [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)\n"
-msgstr "**Alternativ-Download:** [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)\n"
+msgstr "**Mirror 2:** [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)\n"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:20
 #, no-wrap
 msgid "**Install Jamulus**: Double click the installer to launch it. If you get a warning notice from SmartScreen, click on \"More info\" and \"Run anyway\" to install Jamulus. (If you grabbed a new version of Jamulus and are one of the first people who downloaded it, Jamulus won't be whitelisted by SmartScreen yet. We don't pay for code signing.)\n"
-msgstr "**Installiere Jamulus**: Doppelklicke auf Installer, um die Installation zu starten. Wenn du eine Warnung von SmartScreen bekommst, klicke auf „Mehr Infos“ und „Trotzdem ausführen“, um Jamulus zu installieren. (Wenn du eine neue Version von Jamulus heruntergeladen hast und auch einer der Ersten warst, wird Jamulus noch nicht auf der SmartScreen-Whitelist sein. Du erhältst diese Meldung, da wir momentan nicht für ein Software Zertifikat für Jamulus zahlen.)\n"
+msgstr "**Installiere Jamulus**: Doppelklicke auf Installer, um die Installation zu starten. Wenn du eine Warnung von SmartScreen bekommst, klicke auf \"Mehr Infos\" und \"Trotzdem ausführen\", um Jamulus zu installieren. (Wenn du eine neue Version von Jamulus heruntergeladen hast und auch einer der Ersten warst, wird Jamulus noch nicht auf der SmartScreen-Whitelist sein. Du erhältst diese Meldung, da wir momentan nicht für ein Software Zertifikat für Jamulus zahlen.)\n"
```

Comment:
I'd prefer you use the german Gensefüßchen, not "

Reply:
Encountered encoding issues when using "german" „ “ in PO files
@ignotus666 - need instructions on how to encode „ “ correctly in PO files

Reply:
Updated apostrophs